doc/DeveloperGuide.md

```diff
@@ -91,9 +91,10 @@ Use case ends.
 ## Appendix C : Non Functional Requirements
 
 1. Should work on any [mainstream OS](#mainstream-os) as long as it has Java 8 or higher installed.
-2. Should be able to hold up to 1000 persons.
+2. Should be able to hold up to 2000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should have an intuitive UI.
```

Comment:
This is hard to define more precisely what you mean by `intuitively` but should give it a try.  e.g. `Non IT workers who regularly use a computer in their work should be able to understand the UI without having to read a user guide`